### ISSUE TITLE
Running PAML/Codeml on 'treeless' mode.

### ISSUE THREAD

**Issue by kyrgyzbala** [biopython/biopython#964]
Hi guys!

First of all, thank you all for fantastic library.
I was wondering, if there are any plans for enabling PAML/codeml to run without a species tree, wherein it would be possible to parse out pairwise dNdS values only.

If it is not against the policy or something and nobody is planning to do it, I would like to contribute to the code.

Cheers,
Sanjar.

**Comment by peterjc** [comment]
What do you think @brandoninvergo? (Brandon wrote the current PAML support in Biopython).

**Comment by brandoninvergo** [comment]
I didn't know it could run without a species tree.  These programs are full of surprises.  Anyway, I'm not opposed to it being added but I don't have a lot of time at the moment so I would appreciate it if Sanjar could implement it.  The main task will be parsing the output, especially  checking if the output changes significantly for, e.g., different NSsites values.  The tests will also have to be updated; I don't know if this is a newer feature, so you'll have to try it with several older versions of the programs.

I can provide feedback as necessary.

**Comment by kyrgyzbala** [comment]
Yes, parsing the output and making sure it is working right is the biggest issue. 
Making the program run without a species tree is simple though. All it takes is to change the parameter "runmode" from 0 to -2 and skipping the tree file in the argument list. 

OK, I will implement and submit the code in early November, presumably.

Thanks,
Sanjar.

**Comment by zachrodriguez** [comment]
Has this been implemented in biopython @kyrgyzbala?
